shap/_explanation.py

```diff
@@ -803,56 +815,42 @@ def is_1d(val):
     return not (isinstance(val[0], (list, np.ndarray)))
 
 
-class Op:
-    pass
-
-
-class Percentile(Op):
-    def __init__(self, percentile):
-        self.percentile = percentile
-
-    def add_repr(self, s, verbose=False):
-        return "percentile(" + s + ", " + str(self.percentile) + ")"
```

Comment:
Note: this is removed, I don't think this is used anywhere.

Reply:
Perhaps this class was intended as a better way of representing limits in plots. In `plots._utils.parse_axis_limit()`, we currently accept a string like `"percentile(20)"`. I presume the intention was to replace that with a python object.

In any case, I agree it's fine to remove this class for now as it's unused.